src/client/websocket/WebSocketShard.js

```diff
@@ -214,13 +214,12 @@ class WebSocketShard extends EventEmitter {
       }
 
       if (this.connection) {
-        this.debug(`A connection was found. Cleaning up before continuing.
+        this.debug(`A connection object was found. Cleaning up before continuing.
     State: ${CONNECTION_STATE[this.connection.readyState]}`);
-        this._cleanupConnection();
-        this.connection.close(1000);
+        this.destroy({ emit: false });
       }
 
-      const wsQuery = { v: client.options.ws.version };
+      const wsQuery = { v: client.options.ws.version, vladdyUUID: Date.now().toString(36) };
```

Comment:
`vladdyUUID`?

Reply:
Right. Forgot to remove debug variables >.>